src/MoviePageComponent/MoviePageComponent.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import MoviesService from '../api/moviesService';
+import Carousel from '../carousel/Carousel';
+import CastingList from './CastingListComponent';
+import Loading from '../LoadingComponent/LoadingComponent';
+
+const imagePathToSrc = imagePath => `https://image.tmdb.org/t/p/w500/${imagePath}`;
+
+const formatDate = (givenDate, onlyYear = true) => {
```

Comment:
this function is doing too many things, please write separate functions for each case